lib/model/image_sscd.py

```diff
@@ -0,0 +1,60 @@
+from typing import Dict
+import io
+
+# from lib.model.generic_image import GenericImageModel
+from lib.model.model import Model
+from lib import schemas
+from torchvision import transforms
+import torch
+from lib.logger import logger
+import numpy as np
+from PIL import Image
+import urllib.request
+
+imagenet_normalization_mean = [0.485, 0.456, 0.406]
+imagenet_normalization_std = [0.229, 0.224, 0.225]
+class Model(Model):
+    def __init__(self):
+        super().__init__()
+        #FIXME: Load from a Meedan S3 bucket
+        try:
+            self.model = torch.jit.load("sscd_disc_mixup.torchscript.pt")
+        except:
```

Comment:
Can we get a more explicit/focused exception here? What are we rescuing against in this situation?